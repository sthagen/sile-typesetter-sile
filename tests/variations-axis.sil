\begin[papersize=a6]{document}
\use[module=packages.retrograde,target=v0.15.0]
\neverindent
\nofolios
\font[family=Tourney,size=48pt]
\font[variations="wght=150,wdth=120"]{Foo bar}\par
\font[weight=750,variations=wdth=80]{Foo bar}
\end{document}
