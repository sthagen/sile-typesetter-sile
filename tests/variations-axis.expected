Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	56.9764
Set font 	Tourney;48;400;;normal;;wght=150,wdth=120;LTR;
T	68 369 369 w=104.0880	(Foo)
Mx 	138.3008
T	272 244 408 w=101.0400	(bar)
Mx 	14.8819
My 	114.5764
Set font 	Tourney;48;750;;normal;;wdth=80;LTR;
T	68 369 369 w=68.3520	(Foo)
Mx 	95.6771
T	272 244 408 w=72.2880	(bar)
End page
Finish
